components/clinical/medication-summary/src/organisms/medication-summary.tsx

```diff
@@ -0,0 +1,89 @@
+import { FC, HTMLAttributes } from 'react'
+import styled from '@emotion/styled'
+import { CodeSystem, Maybe, MedicationRequest } from '@ltht-react/types'
+import { DateSummary } from '@ltht-react/summary'
+
+import Title from '../atoms/medication-title'
+import Route from '../atoms/medication-route'
+import Indication from '../atoms/medication-indication'
+
+import AdditionalInfo from '../molecules/medication-additional-info'
+import Redacted from '../molecules/medication-redacted'
+import MedicationDosageInstructions from '../molecules/medication-dosage-instructions'
+
+const StyledSummary = styled.div`
+  display: flex;
+  flex: 1;
+  /* flex-basis: 0; */
```

Comment:
can this be deleted?

Reply:
Removed
https://github.com/ltht-epr/ltht-react/pull/30/commits/4d75004bf815499df2faa6058dffa25b11955430